py/src/braintrust/test_logger.py

```diff
@@ -15,10 +14,7 @@
     assert_dict_matches,
     assert_logged_out,
     init_test_logger,
-    simulate_login,
     simulate_logout,
-    with_memory_logger,
-    with_simulate_login,
```

Comment:
grr

Reply:
?

Reply:
these are pytest fixtures that should stick around